src/photos.cs

```diff
@@ -465,7 +488,7 @@ interface PHFetchResultChangeDetails {
 
 	[iOS (8,0)]
 	[TV (10,0)]
-	[Mac (10,13, onlyOn64 : true)]
+	[Mac (10,15, onlyOn64 : true)]
```

Comment:
This looks very strange: Apple retroactively removed the API from previous macOS versions?

Reply:
Apple has that available as Mac 10,15 on xcode, but Alex mentioned to me that we should keep the versions we have so I went back and tried to change them, but I must have missed this one. Will change back to 10,13!

Reply:
Don't blindly trust the headers 😄